source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -204,24 +202,28 @@ Order of Execution
 
 The ``bulkWrite()`` method accepts an optional ``BulkWriteOptions`` as
 a second parameter to specify if you want to execute the bulk operations
-as ordered or unordered. 
+as ordered or unordered.
 
 Ordered Execution
 ~~~~~~~~~~~~~~~~~
 
 By default, the ``bulkWrite()`` method executes bulk operations in
-order. This means that the bulk operations execute in the order you
-added them to the list until an error occurs, if any. 
+order. This means that the operations execute in the order you
+added them to the list until any error occurs.
 
 Example 
 ```````
 
 The following example performs these bulk operations:
 
-- An insert operation for a document where the ``_id`` is "3"
-- A replace operation for a document where the ``_id`` is "1" with a document that contains an additional field
-- An update operation for a document where the ``_id`` is "3" to a document that contains an additional field
-- A delete operation for all documents that contain the field ``x`` with the value "2"
+- An insert operation for a document for a person whose ``name`` is
+  ``"Zaynab Omar"`` and ``age`` is ``37``
```

Comment:
Suggestion: "for a document for a person" is a bit confusing to read. Maybe simplify to something like:
```suggestion
- An insert operation for a document where the ``name`` is
  ``"Zaynab Omar"`` and  the ``age`` is ``37``
```